src/components/ProjectHeader.js

```diff
@@ -5,28 +5,79 @@ import {
   Title,
   PrimaryNav,
   NavMenuButton,
+  NavDropDownButton,
+  Menu,
+  Grid,
+  Button,
 } from "@trussworks/react-uswds";
 import { useTranslation } from "react-i18next";
 
 import LanguageButtons from "./LanguageButtons";
 
-
 const ProjectHeader = ({ currentLanguage, changeLanguage }) => {
   const { t } = useTranslation();
 
   const [expanded, setExpanded] = useState(false);
   const onClick = () => setExpanded((prvExpanded) => !prvExpanded);
 
+  const [isOpen, setIsOpen] = useState([false, false]);
+  const onToggle = (index) =>
+    setIsOpen((prevIsOpen) => {
+      const newIsOpen = [false, false];
+      newIsOpen[index] = !prevIsOpen[index];
+      return newIsOpen;
+    });
+
+  const navMenuItems = [
```

Comment:
Right now, I have both a drop down menu for language selection and standalone buttons - I'll end up choosing one or the other!

Reply:
Preference to choose the drop-down menu.